SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -0,0 +1,245 @@
+  for (int iVar = 0; iVar < n_species_mixture; iVar++){
+    LaminarViscosityPointers[iVar]->SetViscosity(Temperature, Density);
+    laminarViscosity.at(iVar) = LaminarViscosityPointers[iVar]->GetViscosity();
+  }
+
+  for(int i = 0; i < n_species_mixture; i++){
+    for(int j = 0; j < n_species_mixture; j++){
+      phi.push_back(pow((1 + pow(laminarViscosity[i] / laminarViscosity[j],0.5) * pow(molarMasses[j] / molarMasses[i],0.25)),2) / pow(8 * (1 + molarMasses[i] / molarMasses[j]),0.5));
+      wilkeDenumerator += moleFractions[j] * phi[j];
+    }
+    wilkeDenumeratorSum.push_back(wilkeDenumerator);
+    wilkeDenumerator = 0.0;
+    phi.clear();
+    wilkeNumerator.push_back(moleFractions[i] * laminarViscosity[i]);
+    viscosityMixture += wilkeNumerator[i] / wilkeDenumeratorSum[i];
+  }
+  return viscosityMixture;
+}
+
+su2double CFluidScalar::davidsonViscosity(const su2double * const val_scalars){
+  su2double viscosityMixture = 0.0;
+  su2double fluidity = 0.0;
+  su2double E = 0.0;
+  su2double mixtureFractionDenumerator = 0.0;
+  const su2double A = 0.375;
+  std::vector<su2double> mixtureFractions;
+  mixtureFractions.clear();
+
+  for (int iVar = 0; iVar < n_species_mixture; iVar++){
+    LaminarViscosityPointers[iVar]->SetViscosity(Temperature, Density);
+    laminarViscosity.at(iVar) = LaminarViscosityPointers[iVar]->GetViscosity();
+  }
+
+  for(int i = 0; i < n_species_mixture; i++){
+    mixtureFractionDenumerator += moleFractions[i] * sqrt(molarMasses[i]);
+  }
+
+  for(int j = 0; j < n_species_mixture; j++){
+    mixtureFractions.push_back((moleFractions[j] * sqrt(molarMasses[j])) / mixtureFractionDenumerator);
+  }
+
+  for(int i = 0; i < n_species_mixture; i++){
+    for(int j = 0; j < n_species_mixture; j++){
+      E = (2*sqrt(molarMasses[i]) * sqrt(molarMasses[j])) / (molarMasses[i] + molarMasses[j]);
+      fluidity += ((mixtureFractions[i] * mixtureFractions[j]) / (sqrt(laminarViscosity[i]) * sqrt(laminarViscosity[j]))) * pow(E, A);
+    }
+  }
+  return viscosityMixture = 1 / fluidity;
+}
+
+su2double CFluidScalar::wilkeConductivity(const su2double * const val_scalars){
+  std::vector<su2double> phi;
+  std::vector<su2double> wilkeNumerator;
+  std::vector<su2double> wilkeDenumeratorSum;
+  su2double wilkeDenumerator = 0.0;
+  su2double conductivityMixture = 0.0;
+  wilkeDenumeratorSum.clear();
+  wilkeNumerator.clear();
+
+  for (int iVar = 0; iVar < n_species_mixture; iVar++){
+    ThermalConductivityPointers[iVar]->SetConductivity(Temperature, Density, Mu, Mu_Turb, Cp);
+    laminarthermalConductivity.at(iVar) = ThermalConductivityPointers[iVar]->GetConductivity();
+  }
+
+  for(int i = 0; i < n_species_mixture; i++){
+    for(int j = 0; j < n_species_mixture; j++){
+      phi.push_back(pow((1 + pow((laminarViscosity[i]) / (laminarViscosity[j]),0.5) * pow(molarMasses[j] / molarMasses[i],0.25)),2) / pow(8 * (1 + molarMasses[i] / molarMasses[j]),0.5));
```

Comment:
```suggestion
      phi.push_back(pow(1 + sqrt(laminarViscosity[i] / laminarViscosity[j]) * pow(molarMasses[j] / molarMasses[i],1./4),2) / sqrt(8 * (1 + molarMasses[i] / molarMasses[j])));
```

Reply:
I wouldnt use sqrt() over pow(,0.5) for readabilty. And 1./4 is maybe a bit more intuitive than 0.25 (but that is debatable). Removed some superfluous braces as well, please check though